src/main/webapp/i18n/en/lecture.json

```diff
@@ -100,7 +100,16 @@
             "pdfPreview": {
                 "title": "Attachment",
                 "attachmentIDError": "Invalid Attachment or Attachment ID.",
-                "attachmentUnitIDError": "Invalid Attachment Unit or Attachment Unit ID."
+                "attachmentUnitIDError": "Invalid Attachment Unit or Attachment Unit ID.",
+                "removePageButton": "Remove Page(s)",
```

Comment:
You could remove the brackets as they are not present in germen. This is however in my Opinion a matter of taste.

Reply:
Chaged